src/main/jbake/content/documentation/the-sling-engine/sling-api-crud-support.md

```diff
@@ -150,14 +151,31 @@ Delete the resource /myresource
     resourceResolver.delete(myResource);
     resourceResolver.commit();
 
-## Value Class Support
+## Order a Resource
+
+Delete the resource /myresource
```

Comment:
I think this is incorrect.

Reply:
Thanks, fixed in https://github.com/apache/sling-site/pull/72/commits/f4d78988c78b11011821ef638bb6254f3f8cc51a

Reply:
Thanks.